chapters/002-smol-internet/README.md

```diff
@@ -583,21 +478,13 @@ We will add this logic to the `sleep.sh` for chapter 003.
 
 ### What's the difference between `ip addr` and `ip route`?
 
-There's some similar output between the `ip addr` and `ip route` commands. `ip
-addr` gives us view into the network interfaces available on a machine.  `ip
-route` shows us the routing table on that machine.
+There's some similar output between the `ip addr` and `ip route` commands. `ip addr` gives us view into the network interfaces available on a machine.  `ip route` shows us the routing table on that machine.
 
-But it looks like there's routing information in our `ip addr` output? What is
-the difference between a network interface and a routing table?
+But it looks like there's routing information in our `ip addr` output? What is the difference between a network interface and a routing table?
 
-Looking at the output of `ip route`, we see a default gateway identified,
-`default via 10.1.2.1 dev eth0`. This default gateway is what will be used for
-any outgoing packets that are not on the otherwise defined routes. `ip route`
-shows routes on active interfaces. `ip addr` displays all available interfaces
-on a machine, even ones that are not currently active. 
+Looking at the output of `ip route`, we see a default gateway identified, `default via 10.1.2.1 dev eth0`. This default gateway is what will be used for any outgoing packets that are not on the otherwise defined routes. `ip route` shows routes on active interfaces. `ip addr` displays all available interfaces on a machine, even ones that are not currently active. 
 
-`ip route` deals entirely with layer 3 information; whereas `ip addr` has
-information about both layer 2 and layer 3.
+`ip route` deals entirely with layer 3 information; whereas `ip addr` has information about both layer 2 and layer 3.
```

Comment:
I don't know that these words make sense at this point. I don't have a suggestion for it yet, just wanted to comment that there may be some opportunity for another appendix here?